wellregistry/registry/admin.py

```diff
@@ -38,6 +38,11 @@ def _has_permission(perm, user, obj=None):
 
     return user.has_perm(perm) and (not obj or obj.agency.agency_cd in _get_groups(user))
 
+class SelectListFilter(admin.RelatedFieldListFilter):
```

Comment:
I added this after adding USWDS as the long list of agencies was not formatted well and exceeded the length of the page. Googling revealed a simple way to add a different UI element (the select) to implement the list filter.